### PR TITLE
Wdac policy deployment updates

### PR DESCRIPTION
**Pull request by jgeurten**
I am a member of the WDAC application control feature team at Microsoft.

This PR will replace #9495 where the edits were made in the incorrect file.

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Before signing WDAC policies for the first time, be sure to enable rule options
 
 To sign a WDAC policy with SignTool.exe, you need the following components:
 
--   SignTool.exe, found in the Windows SDK (Windows 7 or later)
+-   SignTool.exe, found in the [Windows SDK](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk/) (Windows 7 or later)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md:40]:
```suggestion
-   SignTool.exe, found in the [Windows SDK](https://developer.microsoft.com/windows/downloads/windows-10-sdk/) (Windows 7 or later)
```

File: windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md [hunk 2]
```diff
@@ -47,44 +47,60 @@ If you do not have a code signing certificate, see [Optional: Create a code sign
...
 
-9. Validate the signed file. When complete, the commands should output a signed policy file called DeviceGuardPolicy.bin.p7 to your desktop. You can deploy this file the same way you deploy an enforced or non-enforced policy. For information about how to deploy WDAC policies, see [Deploy and manage Windows Defender Application Control with Group Policy](deploy-windows-defender-application-control-policies-using-group-policy.md).
\ No newline at end of file
+9. Validate the signed file. When complete, the commands should output a signed policy file called {PolicyID}.cip to your desktop. You can deploy this file the same way you deploy an enforced or non-enforced policy. For information about how to deploy WDAC policies, see [Deploy and manage Windows Defender Application Control with Group Policy](deploy-windows-defender-application-control-policies-using-group-policy.md).
+
+
+> [!NOTE]
+   > The device with the signed policy must be rebooted one time with Secure Boot enabled for the UEFI lock to be set.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md:106]:
```suggestion
> The device with the signed policy must be rebooted one time with Secure Boot enabled for the UEFI lock to be set. 
```

File: windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md [hunk 3]
```diff
@@ -47,44 +47,60 @@ If you do not have a code signing certificate, see [Optional: Create a code sign
...
 
-9. Validate the signed file. When complete, the commands should output a signed policy file called DeviceGuardPolicy.bin.p7 to your desktop. You can deploy this file the same way you deploy an enforced or non-enforced policy. For information about how to deploy WDAC policies, see [Deploy and manage Windows Defender Application Control with Group Policy](deploy-windows-defender-application-control-policies-using-group-policy.md).
\ No newline at end of file
+9. Validate the signed file. When complete, the commands should output a signed policy file called {PolicyID}.cip to your desktop. You can deploy this file the same way you deploy an enforced or non-enforced policy. For information about how to deploy WDAC policies, see [Deploy and manage Windows Defender Application Control with Group Policy](deploy-windows-defender-application-control-policies-using-group-policy.md).
+
+
+> [!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md:105]:
```suggestion
   > [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jgeurten, please commit my suggested changes (remove the locale from the link and insert extra spaces).

File: windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md (hunk 3 above)
Comment (illfated) [reply windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md:105]: Good point. The indentation needs to be consistent for all lines in a MarkDown Note blob.

Comment (jgeurten) [reply windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md:105]: It is supposed to be a standalone note; not part of step 9

File: windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md [hunk 4]
```diff
@@ -103,4 +103,4 @@ If you do not have a code signing certificate, see [Optional: Create a code sign
 
 
 > [!NOTE]
-   > The device with the signed policy must be rebooted one time with Secure Boot enabled for the UEFI lock to be set. 
\ No newline at end of file
+   > The device with the signed policy must be rebooted one time with Secure Boot enabled for the UEFI lock to be set.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md:106]:
```suggestion
> The device with the signed policy must be rebooted one time with Secure Boot enabled for the UEFI lock to be set. 
```
(incorrect indentation, as discussed above)

File: windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md (hunk 3 above)
Comment (illfated) [reply windows/security/threat-protection/windows-defender-application-control/use-signed-policies-to-protect-windows-defender-application-control-against-tampering.md:105]: Looks good to me. You may **Resolve** this conversation, if you like.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by jgeurten** [comment]
Good from my side too. Thanks!

**Pull request merged**